nkms/characters.py

```diff
@@ -254,6 +259,15 @@ class Ursula(Character):
     interface = None
     interface_ttl = 0
 
+    def __init__(self, urulsas_keystore=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if urulsas_keystore:
+            self.keystore = urulsas_keystore
+        else:
+            engine = create_engine('sqlite:///:memory:')
+            Base.metadata.create_all(engine)
+            self.keystore = keystore.KeyStore(engine)
```

Comment:
Maybe some warning that we're not persistent here?

Reply:
OK, just pushed a commit that fixes this.